recipes/strling/strling.sh

```diff
@@ -0,0 +1,6 @@
+#bin/bash
```

Comment:
I think this wrapper is not necessary. Conda will usually take care of setting the correct paths.

Reply:
you are right